tests/unit/components/recaptcha-block-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Component | recaptcha-block', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('its actions update its args', function(assert) {
+    const component = this.owner.factoryFor('component:recaptcha-block').create();
```

Comment:
Nice! You mock a class we own, and that's fine.

I will point out that mocking in unit tests, to this extent, makes the mocked class resistent to refactoring (in my experience).